### ISSUE TITLE
No warnings when getter/setter is missing from interface type.

### ISSUE THREAD

**Issue by WORMSS** [microsoft/TypeScript#33116]
**TypeScript Version:**  3.4.0-dev.201xxxxx

**Search Terms:**
type setter no warning, type setter, type getter, type getter setter warning 

**Code**

```ts
interface A {
    readonly prop: boolean;
}

const a: A = {
    prop: false
}

const b: A = {
    get prop() {
        return false;
    }
}
const c: A = {
    set prop(value: boolean) {}
}
a.prop = true;
b.prop = true;
c.prop = true;
```

**Expected behavior:**
variable `b` to show warning that it is an incomplete model,
variable `c` to show warning it's an incomplete model

**Actual behavior:**
no warnings, but runtime errors `Cannot set property prop of #<Object> which has only a getter`

**Playground Link:** 
https://www.typescriptlang.org/play/?target=6#code/JYOwLgpgTgZghgYwgAgILIN4Chm+VCOAEwHsQAbAT2QAcoSaAuZAIxJPMJAG4sBfLFgRkAzmGRxm6ALyYceOg2bxyIiP0HCQY1lOSzseZAHMI4xTQAUASjlGjBMAFcoIZCrW8jAgVp0I9A3lcNXN6KwA3OHInCGY2Di5bDB8sOAA6C31kMChY3hZM8Ozc-KEihhK8iG4gA

**Related Issues:** 

**Follow-up**
I would not expect b to show an error that no setter is supplied if the interface was
```ts
interface A {
  readonly prop: boolean;
}
```
since prop is expected to not have a setter.
And at this point `c` is completely invalid, but again, still no warnings about c object directly, but atleast `Cannot assign to 'prop' because it is a read-only property.` is reported for `a.prop = true`

**ts-config**
```jsonc
{
  "compilerOptions": {
    /* Basic Options */
    "target": "ESNEXT" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
    "strict": true /* Enable all strict type-checking options. */
  }
}
```

**Comment by WORMSS** [comment]
I don't believe this is duplicate. I don't want additional new features. Just want a warning that it's incomplete. If the interface is saying the property is read-write, but the implementation only knows how to read and not write then expect to see an error/warning to say it's incomplete.

**Comment by WORMSS** [comment]
Ok then. The title is very misleading for that then. That a warning for read-only (thing that already exists) would require write-only functionality to be added?

**Comment by WORMSS** [comment]
That seems like a duplicate message, how do I mark the bot as a duplicate and close it's message for house-keeping purposes?

**Comment by WORMSS** [comment]
Seriously, what the fuck is with this bot? Every 3 days, regardless if the task is open or closed, he keeps messaging.. 
How do I Delete the issue completely as if it didn't exist?
